InFlightApp/Views/SettingsPage.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<Page
+    x:Class="InFlightApp.Views.SettingsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:InFlightApp.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+
+    </Grid>
+</Page>
```

Comment:
Zouden we settings niet doen als een popup, dat is normaal gesproken een flyout control (kan je eens in actie zien xaml controls gallery te vinden in windows store)

Reply:
Is voor mij zeker goed!